Mathlib/RingTheory/OreLocalization/Basic.lean

```diff
@@ -96,40 +93,48 @@ variable {R S}
 variable {X} [MulAction R X]
 
 /-- The division in the ore localization `X[S⁻¹]`, as a fraction of an element of `X` and `S`. -/
+@[to_additive "The subtraction in the ore localization,
+as a difference of an element of `X` and `S`."]
 def oreDiv (r : X) (s : S) : X[S⁻¹] :=
   Quotient.mk' (r, s)
 #align ore_localization.ore_div OreLocalization.oreDiv
 
 @[inherit_doc]
 infixl:70 " /ₒ " => oreDiv
 
-@[elab_as_elim]
+@[to_additive (attr := elab_as_elim)]
 protected theorem ind {β : X[S⁻¹] → Prop}
     (c : ∀ (r : X) (s : S), β (r /ₒ s)) : ∀ q, β q := by
   apply Quotient.ind
   rintro ⟨r, s⟩
   exact c r s
 #align ore_localization.ind OreLocalization.ind
 
+@[to_additive]
 theorem oreDiv_eq_iff {r₁ r₂ : X} {s₁ s₂ : S} :
     r₁ /ₒ s₁ = r₂ /ₒ s₂ ↔ ∃ (u : S) (v : R), u • r₂ = v • r₁ ∧ u * s₂ = v * s₁ :=
   Quotient.eq''
 #align ore_localization.ore_div_eq_iff OreLocalization.oreDiv_eq_iff
 
 /-- A fraction `r /ₒ s` is equal to its expansion by an arbitrary factor `t` if `t * s ∈ S`. -/
+@[to_additive "A difference `r -ₒ s` is equal to its expansion by an
+arbitrary factor `t` if `t + s ∈ S`."]
 protected theorem expand (r : X) (s : S) (t : R) (hst : t * (s : R) ∈ S) :
     r /ₒ s = t • r /ₒ ⟨t * s, hst⟩ := by
   apply Quotient.sound
   exact ⟨s, s * t, by rw [mul_smul, Submonoid.smul_def], by rw [← mul_assoc]⟩
 #align ore_localization.expand OreLocalization.expand
 
-/-- A fraction is equal to its expansion by a factor from s. -/
+/-- A fraction is equal to its expansion by a factor from `S`. -/
+@[to_additive "A difference is equal to its expansion by a factor from `S`."]
 protected theorem expand' (r : X) (s s' : S) : r /ₒ s = s' • r /ₒ (s' * s) :=
   OreLocalization.expand r s s' (by norm_cast; apply SetLike.coe_mem)
 #align ore_localization.expand' OreLocalization.expand'
 
 /-- Fractions which differ by a factor of the numerator can be proven equal if
 those factors expand to equal elements of `R`. -/
+@[to_additive "Differences which differ by a factor of the minuend can be proven equal if
```

Comment:
Is "factor" appropriate in the additive setting?

Reply:
Maybe not, but I don't have a better term for it. Do you have any suggestions?

Reply:
"which differ by a common summand"? or "which are translated by a common summand"?